v2g-liberty/rootfs/root/appdaemon/apps/v2g-liberty/modbus_evse_client.py

```diff
@@ -274,20 +273,22 @@ async def initialize(self):
 
     async def initialise_charger(self, v2g_args=None):
         self.log(f"Configuring Modbus EVSE client at {c.CHARGER_HOST_URL}:{c.CHARGER_PORT}, reason: {v2g_args}")
+
+        # Remove old client if needed.
+        if self.client is not None:
+            if self.client.connected:
+                self.client.close()
+            self.client = None
+
         self.client = modbusClient.AsyncModbusTcpClient(
             host=c.CHARGER_HOST_URL,
             port=c.CHARGER_PORT,
-            timeout=3,
-            retries=3,
-            reconnect_delay=500,
-            reconnect_delay_max=1000,
             retry_on_empty=True,
         )
         await self.client.connect()
```

Comment:
I would expect some error handling for the scenario when the `connect()` call fails. Or is that handled somewhere else and did I miss it?

Reply:
Strangely enough the (connection-) errors occur not while connecting but while reading / writing. So that is where the error handling is now is focussed.